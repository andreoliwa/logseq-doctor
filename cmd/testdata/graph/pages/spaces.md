- Regular   text with  spaces
  - child
    - nested [Link   With  Spaces  ](https://example.com)
- top level
  - child 2
  - child 3
    - Here we have #[[some  tag with   spaces]]
