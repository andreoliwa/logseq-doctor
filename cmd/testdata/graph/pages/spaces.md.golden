- Regular   text with  spaces
  - child
    - nested [Link With Spaces ](https://example.com)
- top level
  - child 2
  - child 3
    - Here we have #[[some tag with spaces]]
    - top level 2
- [[some page title with spaces]]
